### ISSUE TITLE
Can't run the theme

### ISSUE THREAD

**Issue by zkvsky** [waldnercharles/Minimis#2]
My Qt version is 5.15.2
Any other theme works.
```
[w] .config/.pegasus-frontend/themes/Minimis-master/theme.qml:112:13: Type ShowcaseView unavailable
[w] .config/pegasus-frontend/themes/Minimis-master/Components/ShowcaseView.qml:54:9: Type ShowcaseCollections unavailable
[w] .config/pegasus-frontend/themes/Minimis-master/Components/ShowcaseCollections.qml:18:5: Type LastPlayedGamesModel unavailable
[w] .config/pegasus-frontend/themes/Minimis-master/Components/Collections/LastPlayedGamesModel.qml:17:9: Cannot assign to non-existent property "delayed"
```

**Comment by waldnercharles** [comment]
This theme is using some cutting edge features of Pegasus. Make sure you're using the latest build. I think it was released 2021-09-26.

**Comment by zkvsky** [comment]
I'm on Gentoo, everything is cutting edge here. Is it even possible to not use latest build? It's being pulled from master is it not?
I pulled build from 4 weeks back and it works, I don't remember which commit now.

**Comment by zkvsky** [comment]
The last commit that even starts the theme is the one from 16 Sep 0e1ac6559af83bb8e32062f4b9c78ea1e65f9339, before adding ShowcaseView, It hard freezes and crashes pegasus though [pegasus-git]

**Comment by waldnercharles** [comment]
Interesting. When I say cutting edge, I meant Pegasus. Do you know which version of Pegasus you're running? And when it was built?

The error from your logs is mentioning that it can't find the "delayed" property, which is a new property available via the latest version of SortFilterProxyModel, built by the latest version of Pegasus.

**Comment by zkvsky** [comment]
~~I think I know the problem, you've tested this theme on Android only didn't you? showcaseViewComponent is Android only I think.
I'm trying to use it on the desktop.~~
I know nothing

I've tested all combos of pegasus, latetst, git, stable, doesn't work/

**Comment by waldnercharles** [comment]
ShowcaseView is just what I happened to call my component.

If you want to fix the problem, you could try just removing line 17 in config/pegasus-frontend/themes/Minimis-master/Components/Collections/LastPlayedGamesModel.qml

It's just there for performance reasons.

**Comment by zkvsky** [comment]
I had to remove it from all the files, where it was used. But it worked!
Thanks!
What's the problem with it though?


P.S I love your theme.

**Comment by zkvsky** [comment]
There's another error 
`[w] file:///home/gregory/.config/pegasus-frontend/themes/Minimis/theme.qml:18:5: QML FontLoader: Cannot load font: "file:///home/gregory/.config/pegasus-frontend/themes/Minimis/assets/fonts/Fontawesome.otf"`

changing it to FontAwesome.otf fixed it

**Comment by waldnercharles** [comment]
That's an interesting one. Maybe windows and android aren't case sensitive. I'll fix that in the repo.

The issue with "delayed" is that it's only available in the latest version of SortFilterProxyModel. I'm not sure why the Linux version doesn't seem to be built with it.

**Comment by zkvsky** [comment]
Thanks.
I miss two things from the previous versions, switching between systems via triggers and some info and video preview on the game screen. I'll hack them on myself if that's not your thing, so it's not a problem.

**Comment by waldnercharles** [comment]
I've talked with mmatyas, and he mentioned that the Linux version of Pegasus should have support for the delayed property.

Just make sure it's latest and not stable.

If it still doesn't work, I'm not sure what the issue could be...

**Comment by zkvsky** [comment]
> I've talked with mmatyas, and he mentioned that the Linux version of Pegasus should have support for the delayed property.
> 
> Just make sure it's latest and not stable.
> 
> If it still doesn't work, I'm not sure what the issue could be...

As I've said already, I've checked with all pegasus versions, my default is built from git master too. And just rechecked it to be sure.
Wouldn't other themes have issues too? Or is it just yours that uses this feature?

**Comment by waldnercharles** [comment]
Mine is likely the only one using the feature right now.

**Comment by waldnercharles** [comment]
Should be fixed with [d0c1688](https://github.com/waldnercharles/Minimis/commit/d0c168862b9bfb033d4c6fcb4014cfd29471ab0d ) and [22dce24](https://github.com/waldnercharles/Minimis/commit/22dce249c3a371f2b45c7f18a64449943cff0727)
